cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -169,11 +170,43 @@ def process_gene_info(gene_info: dict) -> None:
     """
     print("download", gene_info["description"])
     temp_file_path, _ = urllib.request.urlretrieve(gene_info["url"])
+    previous_ref_filepath = os.path.join(env.ONTOLOGY_DIR, f"previous_{gene_info['description']}.csv.gz")
+    os.rename(gene_info["new_file"], previous_ref_filepath)
```

Comment:
can you add a `try-except` block to restore the old file is something fails during this process. I'm imaging something going wrong and how much work it would be to go back and rebuild so we can run the diff again.